packages/app/src/feature-flags/features/index.ts

```diff
@@ -7,88 +7,164 @@ import { EnvironmentConfig } from "App/feature-flags/types"
 import { Feature, Environment } from "App/feature-flags/constants"
 
 export const features: EnvironmentConfig = {
+  // TODO determinate toggle usefulness
```

Comment:
Should it be a scope of another task? If so, maybe we should a task to Jira?

Reply:
https://github.com/mudita/mudita-center/pull/782/commits/94814be14a53bd9a890ccbb1b9372d52aaf5669d